src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -82,35 +91,75 @@ const visitClassDeclaration = (
     classNode.name,
     classNode.typeParameters,
     classNode.heritageClauses,
-    newMembers
+    updatedClassFields
   );
 };
 
-const removeStencilDecorators = (classMembers: ts.ClassElement[]) => {
-  return classMembers.map((m) => {
-    const currentDecorators = m.decorators;
-    const newDecorators = filterDecorators(m, MEMBER_DECORATORS_TO_REMOVE);
+/**
+ * Take a list of `ClassElement` AST nodes and remove any decorators from
+ * method elements which are Stencil-specific decorators. We implement the
+ * intended behavior for these Stencil-specific decorators (things like
+ * `@Watch`, `@PropDidChange`, etc) through a combination of compile- and
```

Comment:
hum, should we remove them while we're in here?

Reply:
I haven't looked at your changes you've made since this comment. If you haven't already, it's probably best we remove them in an explicit commit/PR (e.g. "Remove Deprecated Decorator Names")